reviveme/__init__.py

```diff
@@ -8,6 +9,7 @@ def create_app():
     app.config.from_pyfile("../config.py")
 
     db.init_app(app)
+    migrate = Migrate(app, db)
```

Comment:
qq: was the placement of the migrate variable here as opposed to in `extensions.py` intentional? And if so, how should we decide which file to put new library-related things in in the future?

Reply:
db had to be moved to extensions to avoid circular imports. I think it makes sense to move this to extensions for consistency